packages/themes/src/docs/Description.mdx

```diff
@@ -99,6 +99,33 @@ export const Colors = () => {
 </Story>
 <br />
 
+## Shadows
+
+There are 6 shadow options available as CSS variables on this design system.
+
+export const Shadows = () => {
+  const ShadowsBlock = ({ name }) => (
+    <div className={`${styles[name]} ${styles.shadow}`}>
+      <span className={`${styles.nameShadow}`}>{name}</span>
+    </div>
+  );
+  return (
+    <div className={`${styles.shadowWrapper}`}>
```

Comment:
@esraltintas for this Div and the one on line 108,
Can you either use the components in `.storybook/blocks` or write the styles inline. So that we don't have to put the CSS for the docs in the component CSS file

Reply:
@esraltintas i had a look and i see that there is already documentation only styles in the CSS file. So you can ignore this comment